js/index.js

```diff
@@ -1,7 +1,53 @@
-// Iteration 1: Names and Input
+//1: Names and Input
 
+const hacker1 = "avtandil";
+console.log(`the driver's name is ${hacker1}`);
+const hacker2 = "daniela";
+console.log(`the navigator's name is ${hacker2}`);
+//2: Conditionals
+if (hacker1.length > hacker2.length) {
+  console.log(
+    `The driver has the longest name, it has ${hacker1.length} characters`
+  );
+} else if (hacker1.length < hacker2.length) {
+  console.log(
+    `it seems that the navigator has the longest name, it has ${hacker2.length} characters`
+  );
+} else {
+  console.log(
+    `Wow, you both have equally long names, ${hacker1.length} characters!`
+  );
+}
 
-// Iteration 2: Conditionals
+// Print all the characters of the driver's name, separated by a space and in capitals i.e. "J O H N"
 
+let str = "";
+for (i = 0; i < hacker1.length; i++) {
+  str = str + " " + hacker1[i].toUpperCase();
+}
+console.log(str);
+
+//3.2 Print all the characters of the navigator's name, in reverse order. i.e. "nhoJ"
+hacker2.split("");
+console.log(
+  hacker2
+    .split("")
+    .reverse()
+    .join("")
+);
```

Comment:
nice approach! if you wanted to use a for loop you could also have used something like this
 ```
let str = "";
for (i = 0; i < hacker1.length; i++) {
  str = str + " " + hacker1[i].toUpperCase();
}
console.log(str);
```